contracts/HubPool.sol

```diff
@@ -60,6 +60,8 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         uint256 claimedBitMap;
         // Proposer of this root bundle.
         address proposer;
+        // Keeps track of which SpokePools we have already relayed roots to.
+        mapping(uint256 => bool) relayedRootToSpokePool;
```

Comment:
I think this was left over from your previous implementation, no?

```suggestion
```

Reply:
Removed